docs/fundamentals-how-leakcanary-works.md

```diff
@@ -106,12 +106,12 @@ In the example above, the signature of the leak would be computed as:
 
 ```kotlin
 val leakSignature = sha1Hash(
-    "com.example.leakcanary.LeakingSingle.leakedView" +
+    "com.example.leakcanary.LeakingSingleton.leakedView" +
     "java.util.ArrayList.elementData" +
     "java.lang.Object[].[x]"
 )
 println(leakSignature)
-// d962211eda4d45ac50d88d8f212224116a664968
+// dbfa277d7e5624792e8b60bc950cd164190a11aa
 ```
 
-What's next? Learn how to [fix a memory leak](fundamentals-fixing-a-memory-leak.md)!
\ No newline at end of file
+What's next? Learn how to [fix a memory leak](fundamentals-fixing-a-memory-leak.md)!
```

Comment:
What changed in this line here? Is it a new line being added after this one?

Reply:
Hum, I didn't change this line. Maybe it's an artefact from the Github editor?